addons/website/static/tests/tours/focus_blur_snippets.js

```diff
@@ -0,0 +1,79 @@
+odoo.define(function (require) {
```

Comment:
no name on purpose I guess?
I have always added one, not sure it really is useful except if you need to patch a test in another module tho

Reply:
Always added one in general too but in fact did not see the point for test tour :shrug: I originally created the PR for 15.0 so there is no odoo.define but it's true the module is still reachable via file path in that version. I'll add a name